images:
  rgbname             : 'Database/kinect3rgb.png'
  depthname           : 'Database/kinect3d.png'
  clusteredname       : 'Results/result_cur.png'

clustering:
  depth_weight        : 0.04
  coordinates_weight  : 0
  number_of_clusters  : 6
  depth_thresup       : 80
  depth_thresdown     : 0
  coldict : {
    0: [230, 25, 75],
    1: [60, 180, 75],
    2: [255, 225, 25],
    3: [0, 130, 200],
    4: [245, 130, 48],
    5: [145, 30, 180],
    6: [70, 240, 240],
    7: [240, 50, 230],
    8: [210, 245, 60],
    9: [250, 190, 190],
    10: [0, 128, 128],
    11: [230, 190, 255],
    12: [170, 110, 40],
    13: [255, 250, 200],
    14: [128, 0, 0],
    15: [170, 255, 195],
    16: [128, 128, 0],
    17: [255, 215, 180],
    18: [0, 0, 128],
    19: [128, 128, 128],
    20: [255, 255, 255],
    21: [0, 0, 0]
  }

